src/shared-utils/Miljø.ts

```diff
@@ -48,8 +48,8 @@ const Miljø = (): MiljøProps => {
             dokumentProxyUrl: `https://www.nav.no${basePath}dokument`,
             dokumentUrl: 'http://familie-dokument/familie/dokument/api', //Vil uansett gå til bucket "familievedlegg" enn så lenge,
             modellVersjon: modellVersjon,
-            wonderwallUrl: '',
-            oauthCallbackUri: '',
+            wonderwallUrl: 'https://www.nav.no/oauth2/login?redirect=',
+            oauthCallbackUri: 'https://www.nav.no/oauth2/callback',
```

Comment:
Tror vi er nødt til å ha med mer av pathen her:
```suggestion
            wonderwallUrl: 'https://www.nav.no/familie/barnetrygd/soknad/oauth2/login?redirect=',
            oauthCallbackUri: 'https://www.nav.no/familie/barnetrygd/soknad/oauth2/callback',
```
Eller? 🤔